app/views/record/record.html.erb

```diff
@@ -33,3 +33,31 @@
     <%= render partial: "sidebar" %>
   </div>
 </div>
+
+<% content_for(:head) do %>
+  <link rel="schema.DCTERMS" href="http://purl.org/dc/terms/" />
+  <link rel="schema.DC" href="http://purl.org/dc/elements/1.1/" />
+  <link rel="schema.KEV" href="info:ofi/fmt:kev:mtx:" />
+
+  <meta name="DC.title" content="<%= @record.title %>" />
+  <meta name="DC.type" content="<%= @record.eds_publication_type %>" />
+  <meta name="DC.date" content="<%= @record.eds_publication_year %>" />
+  <% @record.eds_authors.each do |author| %>
+    <meta name="DC.creator" content="<%= author %>" />
+  <% end %>
+  <% if @record.eds_abstract.present? %>
+    <meta name="DCTERMS.abstract" content="<%= @record.eds_abstract %>" />
+  <% end %>
+  <% if @record.eds_source_title.present? %>
+    <meta name="DC.source" content="<%= @record.eds_source_title %>" />
+  <% end %>
+  <meta
```

Comment:
this bit seems to maybe just be for journal articles (at least with the `info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal` value). Since we are addressing type in a separate ticket, I'm not counting this as blocking, but thought I'd comment in case you recall any more info as to whether this was a test or intentional etc